frontend/src/components/worksheets/NewUpload/NewUpload.jsx

```diff
@@ -288,8 +308,8 @@ class NewUpload extends React.Component<{
                 { uploading && <CircularProgressbar
                         className={ classes.progress }
                         variant="determinate"
-                        value={ percentComplete }
-                        text={`${percentComplete}% uploaded`}
+                        value={ parseInt((numeratorComplete / denominatorComplete) * 100) || 0 }

```

Comment:
Can we refactor `parseInt((numeratorComplete / denominatorComplete) * 100) || 0` into a separate variable?